what-is-openstreetmap/index.md

```diff
@@ -5,4 +5,5 @@ order: 0
 color: blue
 description: A brief intro to OpenStreetMap
 image: GOPR7567.jpg
+image-alt: three men talking on a road
```

Comment:
@russbiggs maybe "three men planning a field survey to collect OpenStreetMap data"